admin_tests/nodes/test_views.py

```diff
@@ -0,0 +1,38 @@
+from django.test import RequestFactory
+from nose.tools import *  # flake8: noqa
+
+from tests.base import AdminTestCase
+from tests.factories import NodeFactory
+from admin_tests.utilities import setup_view
+
+from admin.nodes.views import NodeView
+
+
+class TestNodeView(AdminTestCase):
+
+    def test_no_guid(self):
+        request = RequestFactory().get('/fake_path')
+        view = NodeView()
+        view = setup_view(view, request)
+        with assert_raises(AttributeError):
+            view.get_object()
+
+    def test_load_data(self):
+        node = NodeFactory()
+        guid = node._id
+        request = RequestFactory().get('/fake_path')
+        view = NodeView()
+        view = setup_view(view, request, guid=guid)
+        res = view.get_object()
+        assert_is_instance(res, dict)
```

Comment:
Please add a few more checks here to ensure you get back the expected result, not just a `dict`. Likewise for other tests of this form.